src/main/java/com/facebook/openwifirrm/modules/ModelerUtils.java

```diff
@@ -202,4 +211,109 @@ public static double calculateTPCMetrics(
 			return Double.POSITIVE_INFINITY;
 		}
 	}
+
+	private static boolean matchesHtForAggregation(String htOper1, String htOper2) {
+		if (htOper1 == htOper2) { // serves as a null check
+			return true;
+		}
+		HTOperationElement htOperObj1 = new HTOperationElement(htOper1);
+		HTOperationElement htOperObj2 = new HTOperationElement(htOper2);
+		return htOperObj1.matchesForAggregation(htOperObj2);
+	}
+
+	private static boolean matchesVhtForAggregation(String vhtOper1, String vhtOper2) {
+		if (vhtOper1 == vhtOper2) { // serves as a null check
+			return true;
+		}
+		VHTOperationElement vhtOperObj1 = new VHTOperationElement(vhtOper1);
+		VHTOperationElement vhtOperObj2 = new VHTOperationElement(vhtOper2);
+		return vhtOperObj1.matchesForAggregation(vhtOperObj2);
+	}
```

Comment:
these should just be static methods defined in the appropriate classes, not in `ModelerUtils`

Reply:
I call the `matchesForAggregation` which is is in the appropriate class, but since a String is passed in here, I can't move this to the String class. I could just not use a method and instead just to the null check and object creation in the caller of this method. What do you think?

Reply:
I mean you can move these into `HTOperationElement` and `VHTOperationElement` and change to `public`